stunnel/service_defaults.yaml

```diff
@@ -1,20 +0,0 @@
-# Please just read https://www.stunnel.org/static/stunnel.html
```

Comment:
As I said, I'd rather keep it this way. https://www.stunnel.org/static/stunnel.html distinguishes between "global options" and "service level options". We should go with this.